platform/docs/docs/configuration/dataSources/dicom-web.md

```diff
@@ -153,6 +153,18 @@ window.config = {
 };
 ```
 
+##### Data Source Configuration Options
+
+The following properties can be added to the `configuration` property of each data source.
+
+
+- `dicomUploadEnabled`: A boolean indicating if the DICOM upload to the data source is permitted/accepted or not. A value of true provides a link on the OHIF work list page that allows for DICOM files from the local file system to be uploaded to the data source
```

Comment:
I would include a separate header called "DICOM Upload" so that it appears in the indexing for the search of the docs